clusters/veritable-prod/base/flux-system/gotk-sync.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
 spec:
   interval: 1m0s
   ref:
-    branch: main
+    branch: feature/prod-personas
```

Comment:
does this need to be changed back to main before merge?

Reply:
Yes it does, I generally leave it as this until I get a PR approved.